docs/dg/workflow.md

```diff
@@ -43,7 +43,10 @@ The issues for first timers usually have guidance provided in the comment or hav
 * Make sure you know our coding standards.
   {{ embed('Appendix: Coding Standards', 'styleGuides.md', level=2) }}
 * **Follow [the tutorial](https://se-education.org/guides/tutorials/intellijCodeStyle.html) to configure Intellij to follow our coding style**.
-* **This project uses Checkstyle** to check the compliance of Java code. You can use [this document](https://se-education.org/guides/tutorials/checkstyle.html) to find how to use it. In particular, run `gradlew checkstyleMain checkstyleTest checkstyleSystemtest` to check the style of all the relevant Java code.
+* **This project uses Checkstyle** to check the compliance of Java code. You can use [this document](https://se-education.org/guides/tutorials/checkstyle.html) to find how to use it. 
+  * In particular, run `gradlew checkstyleAll` to check the style of all the relevant Java code. This will check the Java code in the following order: main code -> test code -> systemtest code.
+  * If you only want to check the main code, run `gradlew checkstyleMain`. Likewise, for test code, run `gradlew checkstyleTest` and for systemtest code, run `gradlew checkstyleSystemtest`.
+  * You can also run any of the combination of the above, such as `gradlew checkstyleMain checkstyleTest` or `gradlew checkstyleTest checkstyleSystemtest`.
 * **To check Pug files for style errors**, run `npm run lint` from the project root directory. You can use the `npm run lintfix` to automatically fix some of the JavaScript and CSS lint errors.
```

Comment:
@gerhean I think this part is outdated?

Reply:
I don't touch anything regarding the 'compliance of Java code', so I cannot really give a good assessment of whether this part is outdated. As far as I know, the highlighted green parts are still correct.
However style error checks for the Pug portion of the frontend are currently broken, but that isn't the concern of this PR. `npm run lint` also no longer works on the project root directory, but isn't the concern of this PR either.

Reply:
Yup I was referring to the Pug files. Do we have a current working way to lint the frontend?

Reply:
Yes, running `npm run lint` in the `frontend` folder lints the JavaScript.

Reply:
Okay, @yhtMinceraft1010X is it possible to update this line as well? We can change instead of running "from the project root directory", specify that it should be run in the `frontend` folder. We can fix the linting in another PR.

Reply:
You can run `gradlew lintFrontend` too from root, but there is currently no lintfix gradle command.

Reply:
Should `Pug` be changed to `frontend` or `Vue`? We don't have anymore `.pug` files since all the `pug` code is now in `.vue` files.

Reply:
I guess you can say Vue files.